widgets/src/lib/ModelPipelineIcon/ModelPipelineIcon.svelte

```diff
@@ -39,6 +40,7 @@
 		"table-question-answering": IconTableQuestionAnswering,
 		"question-answering": IconQuestionAnswering,
 		"zero-shot-classification": IconZeroShotClassification,
+		"image-to-text": IconImageToText,
```

Comment:
nit: could you add this line after https://github.com/huggingface/huggingface_hub/blob/8bee737a94ee163eb9399e3af1247270c9c550a7/widgets/src/lib/ModelPipelineIcon/ModelPipelineIcon.svelte#L59